src/main/resources/site/parts/RichText/richText.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import '@statisticsnorway/ssb-component-library'
+
+const RichText = ({ text, textType }) => {
+  // Check if text is provided, if not, set default text
+  const displayText = text || 'Skriv her...'
+
+  const processedText = displayText.replace(/<(\/*)p>/gm, '<$1span>')
+
+  let className = ''
+
+  switch (textType) {
+    case 'ingress':
+      className = 'ssb-lead-paragraph'
+      break
+    case 'brodtekst':
+      className = 'ssb-paragraph'
+      break
+    case 'mikrotekst':
+      className = 'ssb-text-wrapper small-text'
+      break
+    default:
+    // Optional: handle the default case if needed
+  }
+
+  return <div className={className} dangerouslySetInnerHTML={{ __html: processedText }} />
```

Comment:
Hvis vi ikke bruker komponentene bør vi enten:
- bruke <p> her istedenfor <div>
eller
- fjerne displayText.replace over

Reply:
støtte på utfordringer med å bruke LeadParagraph, Paragraph og Text small fra SSB-biblioteket, de genererte egne <p>-tagger som genererte feil HTML for render. 
Innkapslingsmetoder: Forskjellige metoder ble utforsket for å omslutte tekst, inkludert bruk av div, p og span for korrekt HTML-output or rendering av text, ende opp med å bruke html methoden fra komponent bibloteket